pkg/client/client_test.go

```diff
@@ -510,7 +511,7 @@ func TestCreateOrUpdateSecret(t *testing.T) {
 
 			c := Client{
 				kclient:       fake.NewSimpleClientset(initial),
-				eventRecorder: events.NewInMemoryRecorder("cluster-monitoring-operator"),
+				eventRecorder: events.NewInMemoryRecorder("cluster-monitoring-operator", clock.RealClock{}),
```

Comment:
Let's use the fake one in tests as suggested here https://github.com/openshift/cluster-policy-controller/pull/148#discussion_r1915489604